frontend/src/libs/components/sidebar/styles.module.scss

```diff
@@ -43,7 +43,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  padding: 14px;
+  padding: 0;
```

Comment:
Do we need this? looks like we can remove it
```suggestion
```

Reply:
@Liza-Veis yep 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/a8e42a57-1376-418e-b7f8-1f9e0879e839)